src/main/java/uk/gov/gchq/palisade/exception/Status.java

```diff
@@ -15,6 +15,8 @@
  */
 package uk.gov.gchq.palisade.exception;
 
+import uk.gov.gchq.palisade.Generated;
+
 import java.util.Arrays;
 
 /**
```

Comment:
use setters for the methods below, (generate setters for statusCode and reason)

Reply:
This is an enum, so would  should expect getters, but not setters.  The equals and hascode will not add any value to the class and I should have added a toString, which I have done now